### DISCUSSION TITLE
How about an embed for Rumble?

### DISCUSSION THREAD

**Discussion by brentarias** [delucis/astro-embed#266] [Feature Requests]
Rumble is the biggest competitor for Youtube.

**Comment by delucis** [comment]
I haven’t heard of Rumble, but if there’s a good way to build an embed component a PR would be welcome.

We have a [guide for how to add a new component](https://github.com/delucis/astro-embed/blob/main/CONTRIBUTING.md#-adding-a-component) that should be helpful. Happy to answer questions if things aren’t clear.

**Comment by nathanchere** [comment]
Programmatic Rumble embedding is unfortunately not as straight-forward as Youtube embedding. I've started on this though, I can't imagine it will take more than a few more days to have something to push.

**Comment by nathanchere** [comment]
I have a working example but can't get it working in astro-embed because of the typescript restrictions clashing with a 'rumble-core' package. I now remember one of the reasons why I always hated typescript so much.

**Comment by delucis** [comment]
> I have a working example but can't get it working in astro-embed

Feel free to open a draft PR and I’ll try to see if I can help out when I have time — thanks!
